src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2421,7 +2421,52 @@
         </Setter>
     </Style>
 
-    <ControlTemplate x:Key="ExpanderToggleButton" TargetType="{x:Type ToggleButton}">
+    <ControlTemplate x:Key="ExpanderCaretToggleButton" TargetType="{x:Type ToggleButton}">
+        <Border Name="Border" 
+                Grid.Row="0" 
+                Background="#4D4D4D"
+                BorderThickness="1" 
+                CornerRadius="1" >
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
+                </Grid.ColumnDefinitions>
+                <Image x:Name="Caret"
+                      Grid.Column="0"
+                      Source="pack://application:,,,/DynamoCoreWpf;component/UI/Images/Caret_Down.png"
```

Comment:
hmm, so this will end up bundling the icons into the assembly right? I think the other icons are just shipped in the UI/images folder - any reason this is better?

Reply:
I chose to `pack` because any other way fails at startup. Not sure why...
The other icons in DynamoModern.xaml are also packed.